src/app/features/ahbs/views/ahb-page/ahb-page.component.spec.ts

```diff
@@ -8,13 +8,28 @@ describe('AhbPageComponent', () => {
     MockBuilder(AhbPageComponent).mock(
       AhbService,
       MockService(AhbService, {
-        getAhb: (params) =>
+        getAhb$Json: jest.fn((params) =>
           of({
             meta: {
               pruefidentifikator: params.pruefi,
+              description: 'Test description',
+              direction: 'Test direction',
+              maus_version: '1.0.0',
             },
+            lines: [],
           }),
-      } as AhbService),
+        ),
+        getAhb$Json$Response: jest.fn(),
+        getAhb$VndOpenxmlformatsOfficedocumentSpreadsheetmlSheet: jest.fn(),
+        getAhb$VndOpenxmlformatsOfficedocumentSpreadsheetmlSheet$Response:
+          jest.fn(),
+        getAhb$Csv: jest.fn(),
+        getAhb$Csv$Response: jest.fn(),
+        getFormatVersions: jest.fn(),
+        getFormatVersions$Response: jest.fn(),
+        getPruefis: jest.fn(),
+        getPruefis$Response: jest.fn(),
+      } as Partial<AhbService>),
```

Comment:
Why mock every single function? This just clutters the code. Please only mock what you actually need.